definition.yml

```diff
@@ -1,9 +1,9 @@
 name: graphicsApiReplacer
 display-name: Graphics API Replacer
-version: 1.2.0
+version: 1.2.1

```

Comment:
If we try to apply semantic versioning, this has a new feature, so it should be `1.3.0`.

Reply:
Addressed

Reply:
Alright